http_request_randomizer/requests/proxy/requestProxy.py

```diff
@@ -51,7 +51,10 @@ def __init__(self, web_proxy_list=[], sustain=False):
         self.parsers = parsers
         self.proxy_list = web_proxy_list
         for i in range(len(parsers)):
-            self.proxy_list += parsers[i].parse_proxyList()
+            try:
+                self.proxy_list += parsers[i].parse_proxyList()
+            except ReadTimeout:
+                self.logger.debug("Proxy Parser failed for: {}".format(parsers[i].url))
```

Comment:
Let's log a warning message instead - it should be something like: self.logger.warn("Proxy Parser: {} TimedOut!".format(parsers[i].url))